### PR TITLE
Update urllib3 to address CVE

### LINKED ISSUE #494: Extpsdk: Update urllib3 version to address CVE
see #493

### PR DESCRIPTION
**Pull request by fhcarter**
Fixes #494 

Update urllib3 requirement in requirements files & update pypi version in build

### CODE REVIEW

**Comment by fhcarter** [comment]
[Jenkins run](https://jenkins.vantiq.com/jenkins/job/vantiq-extension-sources-branch/241/)

File: extpsdk/requirements-sdk.in [hunk 1]
```diff
@@ -1,5 +1,5 @@
 websockets>=12.0
 aiohttp>=3.9.5
-urllib3>=2.0.7
+urllib3>=2.2.2
```
Comment (sfitts) [inline extpsdk/requirements-sdk.in:3]: I don't think we need this part of the change...

**Review by sfitts — CHANGES_REQUESTED** [review]

File: extpsdk/requirements-sdk.in (hunk 1 above)
Comment (fhcarter) [reply extpsdk/requirements-sdk.in]: removed specification of urllib3 altogether.  So should be just in  requirements.txt now.

**Review by sfitts — APPROVED** [review]
LGTM

**Pull request merged**